src/substrate-lib/components/DeveloperConsole/index.ts

```diff
@@ -0,0 +1,33 @@
+// This component will simply add utility functions to your developer console.
```

Comment:
In terms of file system, I suggest creating a `DeveloperConsole.tsx` and using `index.ts` solely for exporting. This allows us to be more assertive when trying to search for files. Example:

With `DeveloperConsole.tsx`:
![image](https://user-images.githubusercontent.com/43269067/191026462-089308e8-2c7c-4f92-98e0-0221319093d2.png)

Without:
![image](https://user-images.githubusercontent.com/43269067/191026879-a819bf64-ee1a-40c0-a276-d449c432393a.png)

Reply:
That makes sense. Currently most of the files have been configured with index. I think we could do refactoring for that later on with a dedicated PR.